### PR TITLE
IP-469: Set database timeout to 200ms by default

### PR DESCRIPTION
**Pull request by gregbeech**
This change automatically set the DB statement timeout to 200ms unless specifically overridden by an environment variable.

This changes the database used by the app to Postgres to the timeout can be queried, and adds a rake task to display it as I couldn't see any other way of executing it in the context of the Rails app (thanks @danielcooper for the idea).

### CODE REVIEW

File: lib/roo_on_rails/railties/database.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module RooOnRails
+  module Railties
+    class Database < Rails::Railtie
+      initializer 'roo_on_rails.database', after: 'active_record.initialize_database' do |_app|
+        $stderr.puts 'initializer roo_on_rails.database'
+
+        ActiveRecord::Base.configurations[Rails.env]['timeout'] = ENV.fetch('DATABASE_TIMEOUT', 200)
```
Comment (mezis) [inline lib/roo_on_rails/railties/database.rb]: Does this actually set the statement timeout for Postgres? Or the connection timeout?

Comment (gregbeech) [inline lib/roo_on_rails/railties/database.rb]: Bugger, I've been looking at entirely the wrong timeout... what an idiot.

Comment (gregbeech) [inline lib/roo_on_rails/railties/database.rb]: @mezis fortunately the approach was sound, it now sets the statement timeout.

**Comment by gregbeech** [comment]
OK so this does work, I've tested it by using the local gem in logistics dashboard and run a `SHOW statement_timeout` command on the ActiveRecord connection. But given sqlite doesn't have the concept of statement timeouts there's no way I can see to test that it's working, unless we use something like postgres as the default database in the test app -- but do we really want to go there?

**Comment by mezis** [comment]
> unless we use something like postgres as the default database in the test app -- but do we really want to go there?

Why not? All the Rails versions we're using can pick up a `DATABASE_URL`, I think*; and Postgres is already on your machine and easy to install in CI?

* if they don't that's maybe a good ticket to add to `roo_on_rails`!

**Comment by gregbeech** [comment]
> Postgres is already on your machine and easy to install in CI?

**Comment by gregbeech** [comment]
See #13 instead.

**Pull request closed without merging**